neuland/simulation/generator/generator.h

```diff
@@ -0,0 +1,137 @@
+#pragma once
+#include <Math/GenVector/Cartesian3D.h>
+#include <Math/GenVector/Polar3D.h>
+#include <Math/GenVector/PxPyPzE4D.h>
+#include <algorithm>
+#include <cmath>
+#include <fmt/format.h>
+#include <TRandom.h>
+
+
+template <typename ADist, typename EDist>
```

Comment:
1. Naming convention. Type should be in CamelCase. Variables should be in small_case.
2. Naming should be self-explanatory. Better use `AngleDist` and `EnergyDist`